docs/decision_record.md

```diff
@@ -1,7 +1,30 @@
 # Architectural Decision Record
 
+## 2021-08-25 - Sequence collection digests will reflect sequence order
+
+### Decision
+
+The final sequence collection digests must reflect the order of the sequences given. In other words, changing the order of the sequences will change the identifier.
```

Comment:
I have a slight issue with the `must reflect order` as I think that the decision to support ordered collections should be left to the implementation.